### PR TITLE
Modelviewer

### PR DESCRIPTION
**Pull request by ArchDemons**

### CODE REVIEW

**Comment by tonihele** [comment]
Nice by the codewise looks of it. This actually helps too my refactoring. At one point I was accepting the fact that ModelViewer is gone at the same time I separate our stuff, I just can't support it. Looks interesting those Creature thingies, need to try it...

**Comment by tonihele** [comment]
Nice, especially secret door is erotic to watch :)

File: src/toniarts/openkeeper/tools/modelviewer/nifty/CreatureControl.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import de.lessvoid.nifty.controls.Parameters;
+import de.lessvoid.nifty.elements.Element;
+import de.lessvoid.nifty.input.NiftyInputEvent;
+import de.lessvoid.nifty.screen.Screen;
+
+/**
+ *
+ * @author archdemon
+ */
+public class CreatureControl implements Controller {
```
Comment (tonihele) [inline src/toniarts/openkeeper/tools/modelviewer/nifty/CreatureControl.java]: This is not used? Maybe better delete

Comment (ArchDemons) [inline src/toniarts/openkeeper/tools/modelviewer/nifty/CreatureControl.java]: For future use

Comment (tonihele) [inline src/toniarts/openkeeper/tools/modelviewer/nifty/CreatureControl.java]: Lol, ok, I'll take it in as a baby you need to nurture in the future :)

**Pull request merged**